From 571522aff0d0f01550ea7a1ebde5531f30ae05bf Mon Sep 17 00:00:00 2001
From: dje <dje@138bc75d-0d04-0410-961f-82ee72b054a4>
Date: Wed, 5 Nov 2014 14:53:23 +0000
Subject: [PATCH 335/341] Backported from mainline. 2014-10-30  David Edelsohn 
 <dje.gcc@gmail.com>

* configure.host (aix4.3+, 5+): Do not use -G in link command.


git-svn-id: svn+ssh://gcc.gnu.org/svn/gcc/branches/gcc-4_8-branch@217131 138bc75d-0d04-0410-961f-82ee72b054a4
---
 libstdc++-v3/ChangeLog      | 7 +++++++
 libstdc++-v3/configure.host | 1 -
 2 files changed, 7 insertions(+), 1 deletion(-)

diff --git a/libstdc++-v3/ChangeLog b/libstdc++-v3/ChangeLog
index 22edbcf..eadf702 100644
--- a/libstdc++-v3/ChangeLog
+++ b/libstdc++-v3/ChangeLog
@@ -1,3 +1,10 @@
+2014-11-05  David Edelsohn  <dje.gcc@gmail.com>
+
+	Backported from mainline.
+	2014-10-30  David Edelsohn  <dje.gcc@gmail.com>
+
+	* configure.host (aix4.3+, 5+): Do not use -G in link command.
+
 2014-10-15  Jason Merrill  <jason@redhat.com>
 
 	* libsupc++/dyncast.cc (__dynamic_cast): Handle mid-destruction
diff --git a/libstdc++-v3/configure.host b/libstdc++-v3/configure.host
index 23b3f52..feae3cc 100644
--- a/libstdc++-v3/configure.host
+++ b/libstdc++-v3/configure.host
@@ -219,7 +219,6 @@ case "${host_os}" in
     os_include_dir="os/aix"
     atomicity_dir="os/aix"
     atomic_word_dir="os/aix"
-    OPT_LDFLAGS="-Wl,-G"
     ;;
   aix4.*)
     os_include_dir="os/generic"
-- 
2.0.3

